armi/physics/neutronics/fissionProductModel/fissionProductModelSettings.py

```diff
@@ -28,29 +29,127 @@ def defineSettings():
             CONF_FP_MODEL,
             default="infinitelyDilute",
             label="Fission Product Model",
-            description="The fission product model to use in this ARMI run",
+            description=(
+                "This setting is used to determine how fission products are treated in an analysis. "
+                "By choosing `noFissionProducts`, no fission products will be added. By selecting, `infinitelyDilute`, "
+                "lumped fission products will be initialized to a very small number on the blocks/components that require them. "
+                "By choosing `MO99`, the fission products will be represented only by Mo-99. This is a simplistic assumption that "
+                "is commonly used by fast reactor analyses in scoping calculations and is not necessarily a great assumption for "
+                "depletion evaluations. Finally, by choosing `explicitFissionProducts` the fission products will be added explicitly "
+                "to the blocks/components that are depletable. This is useful for detailed tracking of fission products."
+            ),
             options=[
                 "noFissionProducts",
                 "infinitelyDilute",
-                "2ndOrder",
-                "2ndOrderWithTransmutation",
                 "MO99",
+                "explicitFissionProducts",
+            ],
+        ),
+        setting.Setting(
+            CONF_FISSION_PRODUCT_LIBRARY_NAME,
+            default="",
+            label="Fission Product Library",
+            description=(
+                f"This setting is used when the `{CONF_FP_MODEL}` setting "
+                f"is set to `explicitFissionProducts` and is used to configure "
+                f"all the nuclides that should be modeled within the core. "
+                f"Setting this is equivalent to adding all nuclides in the "
+                f"selected code library (i.e., MC2-3) within the blueprints "
+                f"`nuclideFlags` to be [xs:true, burn:false]. This option acts "
```

Comment:
I will take a look again at the logic and see if `burn: false` or `burn: true` makes more sense. My reason for not adding `burn: true` is that this is used to determine which regions are depletable in the model. 

The main purpose of this implementation is to allow all the fission products to exist in the block/components that need to be there so that a code like `ORIGEN` could be run and put number densities from its separate solve on the blocks/components. This is not intended to be used with ARMI's built-in `burn-chain.yaml` file since that would then require a user to also build their own `burn-chain.yaml` file. If they want to do this then that is fine, but then they should also go and manually setup the `nuclideFlags` in the blueprints based on what they are trying to do.

Reply:
Nah, I think your implementation with `burn: False` is the right way to go. The aspect about `getDefaultNuclideFlags()` wasn't clear to me at first.

It appears that a case will always have burnable actinides defined, unless the user specifically turns those off.

So I think this is fine.